.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,41 @@
+name: Docker Build and Push
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: "Build latest and versioned docker tags"
+        uses: docker/build-push-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+          repository: deepmind/kapitan
+          add_git_labels: true
+          tag_with_ref: true
+          dockerfile: Dockerfile
+      - name: Strip git ref prefix from tag version and store in TAG_VERSION VAR
+        run: echo ::set-env name=TAG_VERSION::${GITHUB_REF#refs/*/release-}
+      - name: Building/Pushing Tag Version
```

Comment:
Maybe 'Show TAG_VERSION' ? I'm assuming `echo ::set-env name=TAG_VERSION::${GITHUB_REF#refs/*/release-} && echo ${{ env.TAG_VERSION }}` didn't work, otherwise we could remove this step.

Reply:
Yup :) I'll remove that once testing is done to make sure everything looks good first.